Sources/NIOSSL/NIOSSLClientHandler.swift

```diff
@@ -14,17 +14,30 @@
 
 import NIO
 
-private extension String {
-    func isIPAddress() -> Bool {
+internal func validateSNIServerName(_ hostname: String) throws {
+    func isIPAddress(_ string: String) -> Bool {
```

Comment:
If we do then this can go back to being part of an extension on `String` too.

Reply:
sure